spirv-tools-sys/generate.rs

```diff
@@ -0,0 +1,115 @@
+// The spirv tools use generated code, for now we just replicate the minimum
+// generation we need here by calling the *shudders* python script(s) we need
+// to in a simple script and commit them to source control, as they only need
+// to be regenerated when spirv-headers is updated
+
+use std::{fs, process::Command};
+
+fn python<S: AsRef<std::ffi::OsStr>>(args: impl IntoIterator<Item = S>) -> Result<(), i32> {
+    Command::new("python")
```

Comment:
I'm tempted to use PyO3 here to be cute and neat, but considering this is a manual script run by developers and so we can assume python is installed, probably not worth the effort (especially if it gets built by default and so explodes build time due to building PyO3)

Reply:
Yah, would be a bit overkill in this case, but could be fun for someone to PR in the future. :wink: